distribution/dashboard/ui/src/pages/Tenant/Detail/NewBackup/index.tsx

```diff
@@ -0,0 +1,329 @@
+import { getNSName } from '@/pages/Cluster/Detail/Overview/helper';
+import { createBackupPolicyOfTenant, getTenant } from '@/services/tenant';
+import { PageContainer } from '@ant-design/pro-components';
+import { useNavigate } from '@umijs/max';
+import { useRequest, useUpdateEffect } from 'ahooks';
+import {
+  Button,
+  Card,
+  Col,
+  Form,
+  Input,
+  InputNumber,
+  Radio,
+  Row,
+  Select,
+  Space,
+  Switch,
+  TimePicker,
+  message,
+} from 'antd';
+import { clone } from 'lodash';
+import { useState } from 'react';
+import BasicInfo from '../Overview/BasicInfo';
+import type { ParamsType } from './ScheduleSelectComp';
+import ScheduleSelectComp from './ScheduleSelectComp';
+import styles from './index.less';
+const { Password } = Input;
+export default function NewBackup() {
+  const navigate = useNavigate();
+  const [ns, name] = getNSName();
+  const [form] = Form.useForm();
+  const [isExpand, setIsExpand] = useState(false);
+  const [passInputExpand, setPassInputExpand] = useState(false);
+  const [jobInputExpand, setJobInputExpand] = useState(false);
+  const [recoveryInputExpand, setRecoveryInputExpand] = useState(false);
+  const scheduleValue = Form.useWatch(['scheduleDates'], form);
+  const distType = [
+    { label: 'OSS', value: 'OSS' },
+    { label: 'NFS', value: 'NFS' },
+  ];
+
+  const handleSubmit = async (values: any) => {
+    const res = await createBackupPolicyOfTenant(values);
+    if (res.successful) {
+      message.success(res.message, 3);
+      form.resetFields();
+      history.back();
+    }
+  };
+
+  const initialValues = {
+    scheduleDates: {
+      mode: 'Weekly',
+      days: [],
+    },
+  };
+  const { data: tenantDetailResponse } = useRequest(getTenant, {
+    defaultParams: [{ ns, name }],
+  });
+
+  const tenantDetail = tenantDetailResponse?.data;
+
+  useUpdateEffect(() => {
+    let newScheduleValue = clone(scheduleValue);
+    scheduleValue.days.forEach((key: number) => {
+      if (!scheduleValue[String(key)]) {
+        newScheduleValue[key] = 'Full';
+        form.setFieldValue('scheduleDates', newScheduleValue);
+      }
+    });
+    Object.keys(scheduleValue).forEach((key) => {
+      if (/^[\d]+$/.test(key) && !scheduleValue?.days.includes(Number(key))) {
+        delete newScheduleValue[key];
+        form.setFieldValue('scheduleDates', newScheduleValue);
+      }
+    });
+  }, [scheduleValue]);
+
+  return (
+    <PageContainer
+      style={{ paddingBottom: 70 }}
+      header={{
+        title: '创建租户备份策略',
+        onBack: () => navigate(`/tenant/ns=${ns}&nm=${name}/backup`),
+      }}
+      footer={[
+        <Button
+          onClick={() => navigate(`/tenant/ns=${ns}&nm=${name}/backup`)}
+          key="cancel"
+        >
+          取消
+        </Button>,
+        <Button key="submit" onClick={() => form.submit()}>
+          提交
+        </Button>,
+      ]}
+    >
+      <Row style={{ marginBottom: 24 }} gutter={[16, 16]}>
```

Comment:
Do we need a row space? If not, should we use tenant detail&&<Row>

Reply:
Alright, however, the Row needs to be positioned within BasicInfo.